it/uk/gov/hmrc/agentaccesscontrol/support/BaseISpec.scala

```diff
@@ -218,10 +215,18 @@ trait StubUtils {
           """.stripMargin)))
       this
     }
+
+    private def getAssignedAgentsPost(utr: SaUtr) = {
+      post(urlEqualTo(path))
+        .withRequestBody(matching(s".*>$utr<.*"))
+        .withHeader("Content-Type", equalTo("application/xml; charset=utf-8"))
```

Comment:
The way it's coded here makes it so that the stub will return a 404 if the expected header is not present.

It might be easier to diagnose failures if we remove this check here and verify it after the call happened (that way we'll get an explicit "header wasn't sent" failure). We can make this change it if whoever reviews this thinks it's worth it :)